pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "heal"
```

```suggestion
name = "heal-sdk"
```
i think we should rename this so when publishing, the package name would be `heal-sdk`

---

Excellent! Thank you for catching that!